docs/tutorials/send-payjoin-with-pdk.md

```diff
@@ -0,0 +1,372 @@
+            .build_with_additional_fee(
+                bitcoincore_rpc::bitcoin::Amount::from_sat(1),
+                None,
+                bitcoincore_rpc::bitcoin::FeeRate::MIN,
+                true,
+            )
+            .unwrap()
+            .extract_v1()
+            .unwrap();
+        // Step 5. Send the request and receive response
+        // let payjoin_url = pj_uri.extras.e
+        // BITCOIN:BCRT1Q0S724W239Z2XQGSZV6TE96HMYLEDCTX3GDFZEP?amount=0.01&pj=https://localhost:3000
+        //
+        let res = https_agent()
+            .post(req.url.as_str()) // pj_uri.endpoint
+            .header("Content-Type", "text/plain")
+            .body(String::from_utf8(req.body).unwrap())
+            .send()
+            .await
+            .unwrap();
+        let res = res.text().await.unwrap();
+        // Step 6. Process the response
+        //
+        // An `Ok` response should include a Payjoin Proposal PSBT.
+        // Check that it's signed, following protocol, not trying to steal or otherwise error.
+        let psbt = ctx.process_response(&mut res.as_bytes()).unwrap();
+        // Step 7. Sign and finalize the Payjoin Proposal PSBT
+        //
+        // Most software can handle adding the last signatures to a PSBT without issue.
+        let psbt = sender_wallet
+            .wallet_process_psbt(&base64::encode(psbt.serialize()), None, None, None)
+            .unwrap()
+            .psbt;
+        let tx = sender_wallet
+            .finalize_psbt(&psbt, Some(true))
+            .unwrap()
+            .hex
+            .unwrap();
+        // Step 8. Broadcast the Payjoin Transaction
+        let txid = sender_wallet.send_raw_transaction(&tx).unwrap();
+        txid
+    }
+}
+
+
+```
+
+And lets call the sender functions in the main function:
+
+```
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn std::error::Error>> {
+    setup_wallets();
+    let args: Vec<String> = std::env::args().collect();
+    let payjoin_uri: String = args[1].parse().unwrap();
+    let payjoin_uri = payjoin::Uri::try_from(payjoin_uri)
+        .unwrap()
+        .assume_checked();
+    let txid = Sender::try_payjoin(payjoin_uri).await;
+    dbg!(&txid);
+    Ok(())
+}
+```
+
+# 5. Receiver
+ To setup the receiver, we will use `payjoin-cli` a cli payjoin
+ implementation developed by the `rust-payjoin` team. We will setup
+ this receiver, generate BiP21 URI and go back to the `sender` we
+ implemented and give it the URI.
+
+ Open your terminal, and:
+
+ 1. Install `payjoin-cli`
+
+ ```shell
+ cargo install payjoin-cli --version 0.0.2-alpha --features local-danger-https
+ ```
+
+ 2. Run `payjoin-cli` as the receiver
+
+ *Here we assume that you completed all of the previous steps
+ successfully and generated the `receiver` wallet*.
+
+ ```rust
+  payjoin-cli -r http://localhost:18443/wallet/reciever --rpcuser foo --rpcpass qDDZdeQ5vw9XXFeVnXT4PZ--tGN2xNjjR4nrtyszZx0= receive --endpoint https://localhost:3000 1230000000
```

Comment:
A few things could simplify this manual setup

    Writing a script that automatically runs a payjoin-cli receiver in the docker container
    using a simple password like "insecurepassword"
    using the default cookie auth and having the container output the cookie location on startup

Looks like the script also created a wallet called receiver while this command wants a wallet called reciever (the former is common english)

remove the indentation